en/config_mc/position_mode_smooth.md

```diff
@@ -0,0 +1,91 @@
+# Smooth and Sport Modes
+
```

Comment:
1. "Modes" feels like an overloaded term, when we are in position mode or some other mode already.  Is there some other acceptable way to name these? My preference would be to refer to these as configurations - ie Smooth and Sport Configurations. Other options - trajectory, path, options, model, scheme, config. 
2. It would be good here to have a paragraph or sentence explaining what smooth and sports modes "deliver" - perhaps by contrasting to Jerk limited modes. What differentiates smooth from sport?

Reply:
What about "Slew-rate type" (instead of smmoth and sport) and "Jerk-limited type" ? I had smooth and sport in the same file because the implementation is actually the same and "sport" just uses a few different parameters; but isn't really used ...

Reply:
"Type" works well too - I'm not sure it is better than "config" but it is certainly no worse (and lots better than mode.

Will leave these fixes to you!